eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ledger.hs

```diff
@@ -459,6 +482,22 @@ ledgerTransition = do
         )
   pure $ LedgerState utxoState'' certStateAfterCERTS
 
+processBadWithdrawalsLEDGER ::
+  forall (someLEDGER :: Type -> Type) era.
+  PredicateFailure (someLEDGER era) ~ ConwayLedgerPredFailure era =>
+  Maybe (Map.Map RewardAccount Coin, Map.Map RewardAccount Coin) ->
+  Rule (someLEDGER era) 'Transition ()
+processBadWithdrawalsLEDGER badWithdrawals = do
+  let (invalidWithdrawals, incompleteWithdrawals) =
```

Comment:
This constraint essentially restricts `someLEDGER` to `ConwayLEDGER`. Currently there is no reuse of `ledgerTransition` where `someLEDGER` is other than `ConwayLEDGER`, begs the question of whether it was made polymorphic expecting reuse in Dijkstra? Will we benefit from making it monomorphic?

Reply:
I totally mistook what we discussed on Friday perhaps :disappointed: now that I think more about it.